sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -1348,20 +1359,104 @@ void gnrc_netif_default_init(gnrc_netif_t *netif)
 #endif
 }
 
+#if IS_USED(MODULE_GNRC_NETIF_EVENTS)
+/**
+ * @brief   Call the ISR handler from an event
+ *
+ * @param[in]   evp     pointer to the event
+ */
+static void _event_handler_isr(event_t *evp)
+{
+    (void)evp;
```

Comment:
Why this? `evp` is used just in the line below.

Reply:
Ah, leftover from shuffling around the `#ifdef`s, will remove